### DISCUSSION TITLE
Testing - (int)client_sockets.size() always 0

### DISCUSSION THREAD

**Discussion by ytrewq271828** [ANLAB-KAIST/KENSv3#183] [Project Q&A 2. Transport Layer (TCP)]
Hello. 
I was testing my implementation with test-kens-handshake.
I read through the testhandshake.cpp for debugging.

For every test cases, the line 91 and 167 of testhandshake.cpp always fails.
The value of (int)client_sockets.size() is always 0.
Also there are no more EXPECTED_EQ failures, which makes it difficult to debug.

I will be grateful if you let me know what causes the zero value.
Thanks in advance.

**Comment by Pusnow** [comment]
> The value of (int)client_sockets.size() is always 0.

This means there is no successful `accept()` call. Please check you have implemented `accept()` and `connect()` correctly.
